mobile/src/services/user-details-api/user-details-api.service.ts

```diff
@@ -43,6 +44,26 @@ class UserDetailsApi {
       },
     );
   }
+
+  public updateUserAvatar({
+    file,
+    userId,
+  }: UserDetailsUpdateAvatarRequestDto): Promise<UserDetailsResponseDto> {
+    const { uri, type, fileName: name } = file;
+    const formData = new FormData();
+    formData.append('file', { uri, type, name } as any);
```

Comment:
Avoid `any`

Reply:
Any ideas how i can avoid `any` type here ?

Reply:
Did you try 'string'?

Reply:
Yes I tried `string` and other type. Read your article so far without success

Reply:
I can do it's like this `formData.append('file', { uri, type, name } as unknown as string | Blob);`  but i think it's the same as `any` type